

#print(contador)


try:
    print(contador)
except:
    print("Ocurrio un error")